src/complexity_measures/missing_dispersion.jl

```diff
@@ -0,0 +1,73 @@
+using DelayEmbeddings
+export MissingDispersionPatterns
+
+"""
+    MissingDispersionPatterns <: ComplexityMeasure
+    MissingDispersionPatterns(; est = Dispersion())
+
+An estimator for the number of missing dispersion patterns (``N_{MDP}``), a complexity
+measure which can be used to detect nonlinearity in time series (Zhou et al.,
+2022)[^Zhou2022], using the dispersion pattern estimator `c`.
+
+Used with [`complexity`](@ref) or [`complexity_normalized](@ref)`.
+
+## Description
+
+``N_{MDP}`` is computed by first symbolising each `xᵢ ∈ x`, then embedding the resulting
+symbol sequence (using `est`), and computing the quantity
+
+```math
+N_{MDP} = L - N_{ODP},
+```
+
+where `L = alphabet_length(est)` (i.e. the total number of possible dispersion patterns),
+and ``N_{ODP}`` is defined as the number of *occurring* dispersion patterns.
+
+The normalized complexity measure ``N_{MDP}^N = (L - N_{ODP})/L`` is
+computed. The authors recommend that
```

Comment:
Is it though? it's not computed in the source.

Reply:
I updated the docstring to reflect that this only happens when calling `complexity_normalized`.